src/app/content/components/Content.tsx

```diff
@@ -148,29 +159,32 @@ const HideOverflowAndRedoPadding = isOpenConnector(styled.div`
  *   of things need to know when the sidebar is open/closed.
  */
 // tslint:disable-next-line:variable-name
-const Content: React.SFC = () => <Layout>
-  <Background>
-    <BookBanner/>
-    <Toolbar />
-    <Wrapper>
-      <CenteredContentRow>
-        <Sidebar />
-        <ContentPane>
-          <UndoPadding>
-            <MainContentWrapper>
-              <ContentNotifications />
-              <HideOverflowAndRedoPadding>
-                <Page />
-                <PrevNextBar />
-              </HideOverflowAndRedoPadding>
-              <Attribution />
-              <Footer/>
-            </MainContentWrapper>
-          </UndoPadding>
-        </ContentPane>
-      </CenteredContentRow>
-    </Wrapper>
-  </Background>
-</Layout>;
+const Content: React.SFC = () => <React.Fragment>
+  <GlobalStyle />
```

Comment:
can't you put the globalstyle inside the layout and not have the `React.Fragment` thing?

Reply:
OK I'll do that.